packages/blockchain/xdc/src/lib/xdc.util.ts

```diff
@@ -0,0 +1,10 @@
+export const xdcUtil = {
+  toHex: (address: string) => {
+    if (address.startsWith('xdc')) return `0x${address.substring(3)}`
```

Comment:
or maybe even better to use lodash, trimStart. as you wish.
btw. there is no usage of this util, do we need it?

Reply:
It's used in the xdc service on core-api